src/legacy/ui/public/inspector/ui/inspector_panel.js

```diff
@@ -84,28 +85,30 @@ class InspectorPanel extends Component {
     const { selectedView } = this.state;
 
     return (
-      <React.Fragment>
-        <EuiFlyoutHeader hasBorder>
-          <EuiFlexGroup
-            justifyContent="spaceBetween"
-            alignItems="center"
-          >
-            <EuiFlexItem grow={true}>
-              <EuiTitle size="s">
-                <h1>{ title }</h1>
-              </EuiTitle>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <InspectorViewChooser
-                views={views}
-                onViewSelected={this.onViewSelected}
-                selectedView={selectedView}
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiFlyoutHeader>
-        { this.renderSelectedPanel() }
-      </React.Fragment>
+      <I18nProvider>
```

Comment:
`I18nProvider` should wrap react app's root component (inside each react render method). So the best place is here https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/flyout/flyout_session.tsx#L106

Reply:
Ah, thanks. Fixed.